### PR TITLE
Go live! Deploy! Continuously Deliver that shit!

### PR DESCRIPTION
**Pull request by klaseskilson**
[It's the future](https://circleci.com/blog/its-the-future/)

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def home
 
   def admin
     @in_stock = Drink.where(instock: true).count
-    @unconfirmed_bookings = Booking.where(['confirmed = ? and date > ?', true, DateTime.now]).count
+    @unconfirmed_bookings = Booking.where(['confirmed = ? and date > ?', false, DateTime.now]).count
```
Comment (klaseskilson) [inline app/controllers/static_pages_controller.rb]: :fu:

**Comment by klaseskilson** [comment]
Shall we?

**Comment by danielronnkvist** [comment]
Yes 🔥

**Comment by klaseskilson** [comment]
OK! 🙈

**Pull request merged**